config/locales/en.yml

```diff
@@ -485,6 +491,11 @@ en:
       comment: Comment
       newer_comments: "Newer Comments"
       older_comments: "Older Comments"
+  doorkeeper:
+    flash:
+      applications:
+        create:
+          notice: Application Registered.
```

Comment:
Was there a reason for pulling this in from the upstream translations? I don't think we use it directly do we, though the doorkeeper controllers that we inherit from do.

Reply:
It's part of the created vs registered thing - I wanted to keep the verb consistent between the page titles, action buttons and flash messages.